modules/caddyhttp/replacer.go

```diff
@@ -398,6 +403,20 @@ func getTLSPeerCert(cs *tls.ConnectionState) *x509.Certificate {
 	return cs.PeerCertificates[0]
 }
 
+type requestId struct {
```

Comment:
```suggestion
type requestID struct {
```

This still needs to be updated :+1: then we can merge it. (Make sure to update the name everywhere.)